### ISSUE TITLE
How is surface tension defined for MonteCarloMembraneBarostat?

### ISSUE THREAD

**Issue by moeenmeigooni** [openmm/openmm#4222]
I'm trying to use the membrane barostat in OpenMM 7.7.0 for isotropic XY pressure coupling and separate Z pressure coupling. When I set the surface tension to a negative value and I get an error:

```
barostat = MonteCarloMembraneBarostat(1*atmosphere, , 300*kelvin, -1, 
                                      MonteCarloMembraneBarostat.XYIsotropic,
                                      MonteCarloMembraneBarostat.ZFree)
```
> openmm.OpenMMException: Surface tension cannot be negative

Why can the surface tension not be negative? My understanding is that surface tension in membrane simulations is [conventionally defined](https://livecomsjournal.org/index.php/livecoms/article/view/v1i1e5966) as follows: 

$$ \gamma = \frac{1}{2} L_{z} ( P_{z} - P_{xy} ) $$

With this definition, if $P_{xy} > P_{z}$, the surface tension can be negative; positive surface tension 'expands' the system in the xy plane, while negative surface tension 'contracts' the system size in the xy plane. OpenMM telling me the value cannot be negative leads me to believe that the above equation is not how OpenMM defines surface tension. On the [user guide page](http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#montecarlomembranebarostat) for the membrane barostat, I see an equation that defines the Monte Carlo criterion for accepting/rejecting a volume change, but no further explanation on how surface tension is defined.

How does OpenMM define surface tension?

**Comment by peastman** [comment]
You're right, there's no reason to limit it to positive surface tension.  We should remove that check.

The check is only performed when setting the default surface tension, which is the initial value for newly created contexts.  Once a context is created, it won't stop you from setting a negative value.

```python
context.setParameter(MonteCarloMembraneBarostat.SurfaceTension(), tension)
```

**Comment by peastman** [comment]
The fix is in #4223.

**Comment by moeenmeigooni** [comment]
Great, thank you Peter!
